troubleshooting-1.Rmd

```diff
@@ -24,14 +26,18 @@ Now, let's load the `tidyverse` (meta-) package:
 
 ```{r, error = TRUE}
 ## ERROR 2 ##
-libraries(tidyverse)
+# Line 30 Max: Wrong function for accessing the package, should be library()
```

Comment:
Could add comment for clarity: We changed "libraries(tidyverse)" to the correct call: "library(tidyverse)".

Reply:
changed have a look in the new code.